src/UnsetManyCapableTrait.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: xedin
+ * Date: 1/25/2018
+ * Time: 6:38 PM.
+ */
+
+namespace Dhii\Data\Object;
+
+use OutOfBoundsException;
+use OutOfRangeException;
+use stdClass;
+use Traversable;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use InvalidArgumentException;
+
+/**
+ * Functionality for unsetting multiple keys at once.
+ *
+ * @since [*next-version*]
+ */
+trait UnsetManyCapableTrait
+{
+    /**
+     * Unset data by keys.
+     *
+     * @since [*next-version*]
+     *
+     * @param string[]|int[]|float[]|bool[]|Stringable[]|stdClass|Traversable $keys The keys of data to unset.
+     *
+     * @throws InvalidArgumentException If the list of keys is invalid.
+     * @throws OutOfRangeException      If one of the keys is invalid.
+     * @throws OutOfBoundsException     If one of the keys does not exist.
+     */
+    protected function _unsetMany($keys)
+    {
+        $keys = $this->_normalizeIterable($keys);
+
+        foreach ($keys as $_idx => $_key) {
+            try {
+                $this->_unsetData($_key);
```

Comment:
Abstract method `_unsetData()` is not declared in this trait.

Reply:
Good catch.